test/gbk-test.js

```diff
@@ -2,25 +2,25 @@
 
 var fs = require("fs"),
     assert = require("assert"),
-    Buffer = require("safer-buffer").Buffer,
-    iconv = require("../");
+    utils = require("./utils"),
+    iconv = utils.requireIconv();
 
 var testString = "中国abc", //unicode contains GBK-code and ascii
-    testStringGBKBuffer = Buffer.from([0xd6, 0xd0, 0xb9, 0xfa, 0x61, 0x62, 0x63]);
+    testStringGBKBuffer = utils.bytes("d6 d0 b9 fa 61 62 63");
 
 describe("GBK tests", function () {
     it("GBK correctly encoded/decoded", function () {
         assert.strictEqual(
-            iconv.encode(testString, "GBK").toString("binary"),
-            testStringGBKBuffer.toString("binary")
+            utils.hex(iconv.encode(testString, "GBK")),
+            utils.hex(testStringGBKBuffer)
         );
         assert.strictEqual(iconv.decode(testStringGBKBuffer, "GBK"), testString);
     });
 
     it("GB2312 correctly encoded/decoded", function () {
         assert.strictEqual(
-            iconv.encode(testString, "GB2312").toString("binary"),
-            testStringGBKBuffer.toString("binary")
+            utils.hex(iconv.encode(testString, "GB2312")),
+            utils.hex(testStringGBKBuffer)
```

Comment:
In my understanding changing from `binary` to `hex` should be ok. Just wanted to double check with you.

Reply:
yep you're right.